src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1039,7 +1039,9 @@ private void resolveToken(ref CORINFO_RESOLVED_TOKEN pResolvedToken)
 
 #if READYTORUN
             TypeDesc owningType = methodIL.OwningMethod.GetTypicalMethodDefinition().OwningType;
-            bool recordToken = _compilation.NodeFactory.CompilationModuleGroup.VersionsWithType(owningType) && owningType is EcmaType;
+            bool recordToken = _compilation.NodeFactory.CompilationModuleGroup.VersionsWithType(owningType) &&
+                owningType is EcmaType &&
+                methodIL.GetMethodILDefinition() is EcmaMethodIL; // Don't record pseudo-tokens generated for IL stubs
```

Comment:
Why do we need this? Wouldn't `HandleToModuleToken` make sure we don't record something dumb?

(Trying to see if we can avoid leaking the handling of the faux `MethodIL` into this method.)

Reply:
Removed in 2nd commit - I think this just my initial naive attempt at fixing what ended up as the HandleToModuleToken modification.